tooling/nargo_cli/tests/snapshots/compile_success_empty/regression_7038_2/execute__tests__expanded.snap

```diff
@@ -12,7 +12,7 @@ trait CurveParamsTrait<BigNum>
 where
     BigNum: BigNumTrait,
 {
-    fn one() {}
+    fn one();
```

Comment:
Where is this change from?
I don't see the source file being changed from `fn one() {}` so I'm not sure why the output here would be.

Reply:
I had to fix the unit test `unquoted_integer_as_integer_token()`, the new result looks ok to me.